PublishedModules/mjlill/Astronauts/Astronauts.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+import json
+import requests
+
+import core.base.Managers as managers
+from core.base.model.Intent import Intent
+from core.base.model.Module import Module
+from core.dialog.model.DialogSession import DialogSession
+
+
+class Astronauts(Module):
+	"""
+	Author: mjlill
+	Description: Inquire a list of names of current astronauts
+	"""
+
+	_INTENT_ASTRONAUTS = Intent('Astronauts')
+
+	def __init__(self):
+		self._SUPPORTED_INTENTS	= [
+			self._INTENT_ASTRONAUTS
+		]
+
+		super().__init__(self._SUPPORTED_INTENTS)
+
+	def onMessage(self, intent: str, session: DialogSession) -> bool:
+		if not self.filterIntent(intent, session):
+			return False
+
+		sessionId = session.sessionId
+
+		
+		if intent == self._INTENT_ASTRONAUTS:
+			try:
+				url = 'http://api.open-notify.org/astros.json'
+	
+				req = requests.get(url=url)
+				data = req.json
+	
+				amount = data['number']
+				if not amount:
+					text = managers.TalkManager.getrandomTalk('noAstronauts')
+				elif amount == 1:
+					text = managers.TalkManager.randomTalk(module=self.name, talk='oneAstronaut').format(
```

Comment:
It's correct but if you want to be faster you can use the helper self.randomTalk()

Reply:
ah ok will use that

Reply:
@Psychokiller1888 can you check whether I use it the correct way?